src/app/index.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { hot } from 'react-hot-loader';
+import './app.scss';
+import * as mac from 'assets/imgs/mac.png';
+import * as windows from 'assets/imgs/windows.png';
+import * as linux from 'assets/imgs/linux.png';
+import * as Secure from 'assets/imgs/icon-shield.svg';
+import * as Offline from 'assets/imgs/icon-offline.svg';
+import * as Yours from 'assets/imgs/icon-hdd.svg';
+import * as OpenSource from 'assets/imgs/icon-github.svg';
+import { Block } from './block';
+import { Platform } from './platform';
+
+const builds = [
+  { name: 'Mac', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'Windows', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'Linux', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'HTML Build', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' }
+];
+
+let OS: string = '';
+if (navigator.appVersion.includes('Win')) {
+  OS = 'Windows';
+} else if (navigator.appVersion.includes('Mac')) {
+  OS = 'Mac';
+} else if (navigator.appVersion.includes('Linux')) {
+  OS = 'Linux';
+} else if (navigator.appVersion.includes('X11')) {
+  OS = 'Unix';
+}
+const img: any = OS === 'Mac' ? mac : OS === 'Windows' ? windows : linux;
+
+const App = () => (
+  <div className="App">
+    <main className="App-main">
+      <div className="App-main-left">
+        <h1 className="App-title">MyCrypto for {OS}</h1>
+        <p className="App-description">
+          Run MyCrypto securely and offline with the desktop application. Powered by Electron, it's
+          the same MyCrypto app, running in a sandboxed environment.
+        </p>
+        <a
+          href="https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC"
+          className="App-download"
+        >
+          <i className="nc-icon nc-hit-down" />
+          Download Alpha
+        </a>
+        <br />
+        <div className="App-other-links">
+          <a
+            href="https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC"
+            className="App-other-links-release-notes"
+          >
+            release notes
+          </a>
+          <a
+            href="https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC"
+            className="App-other-links-release-notes"
+          >
+            signatures
+          </a>
```

Comment:
Not sure I'd know what this refers to with this text alone. Maybe "validate builds" or something that links directly to the part of the release / a KB article that explains how to do so.